Makefile

```diff
@@ -21,7 +30,37 @@ fmt-check:
 	@opa fmt . --list --fail >/dev/null 2>&1
 
 # For convenience. If this passes then it should pass in GitHub
-ci: fmt-check test
+ci: fmt-check quiet-test
+
+# Assume you have the build-definitions repo checked out close by
+#
+THIS_DIR=$(shell git rev-parse --show-toplevel)
+BUILD_DEFS=$(THIS_DIR)/../build-definitions
+BUILD_DEFS_SCRIPTS=$(BUILD_DEFS)/appstudio-utils/util-scripts
+DATA_DIR=$(THIS_DIR)/data
+
+define BD_SCRIPT
+.PHONY: $(1)-$(2)
+$(1)-$(2):
+	@cd $(BUILD_DEFS_SCRIPTS) && env DATA_DIR=$(DATA_DIR) ./$(1)-ec-data.sh $(2) $(3)
+endef
+$(eval $(call BD_SCRIPT,fetch,,$(PR)))
```

Comment:
This is a bit "creative" - It's so I can reuse the template for fetch-ec-data and show-ec-data. $(2) will be empty when calling fetch-ec-data.sh .